binds/python/bind_immutable.cpp

```diff
@@ -53,6 +53,10 @@ static void bind_immutable_module(py::module &m) {
                 return py::array(static_cast<py::ssize_t>(morpho->points().size()), morpho->points().data());
             },
             "Returns a list with all points from all sections")
+        .def_property_readonly("section_offsets", [](morphio::Morphology* morpho){
+                return py::array(static_cast<py::ssize_t>(morpho->sectionOffsets().size()), morpho->sectionOffsets().data());
+            },
+            "Returns a list with offsets to access data of a specific section in the points and diameters arrays")
```

Comment:
it would probably be helpful to describe this better - not everyone is used to the `h5` style encoding.  Maybe say that the `n`'th section is at `n` in the array, and that its points range from `points[section_offsets[n], section_offsets[n + 1] - 1]` , that the end has to be manually computed, and whether or not the parent point is included...